RealmSwift/Util.swift

```diff
@@ -191,3 +191,18 @@ internal protocol AssistedObjectiveCBridgeable {
     static func bridging(from objectiveCValue: Any, with metadata: Any?) -> Self
     var bridged: (objectiveCValue: Any, metadata: Any?) { get }
 }
+
+// MARK: Dictionary
+
+extension Dictionary {
+    // map [K0: V0] to [K1: V1] using closure
+    // let dict: [String: Int] = ["a": "1", "b": "2"].map { ($0, Int($1)!) }
+    func map<U, V>(_ closure: (Key, Value) -> (U, V)) -> [U: V] {
```

Comment:
Where is this being used? If it's only in one spot, I would rather not be adding methods directly onto `Dictionary`.

Reply:
will remove from here